core/net/mac/tsch/tsch-private.h

```diff
@@ -112,4 +112,8 @@ void tsch_disassociate(void);
 #define BUSYWAIT_UNTIL_ABS(cond, t0, offset) \
   while(!(cond) && RTIMER_CLOCK_LT(RTIMER_NOW(), (t0) + (offset))) ;
 
+#ifndef CLOCK_LT
+#define CLOCK_LT(a, b) ((int32_t)((a)-(b)) < 0)
+#endif
```

Comment:
I'm afraid this breaks things. `CLOCK_LT` is not standard in Contiki, platforms don't define it (yet?). Therefore, what happens with platforms that don't have `int32_t` as clock type?